source/frontend/src/features/notes/list/NoteListView.tsx

```diff
@@ -93,70 +92,66 @@ export const NoteListView: React.FunctionComponent<React.PropsWithChildren<INote
   };
 
   return (
-    <ListPage>
```

Comment:
Hmm, is this related to the other deposit changes? Is there another jira issue that this is in reference to?

Reply:
Not an issue with deposits. It's a padding issue with the notes in all screens that use the widget. Figure it would be ok to piggy back it with the UI changes.